microsoftgraph/microsoft-graph-explorer-v4
# Fix: Focus order from Modify Permissions tab to Access Token tab

## Overview

Fixes #1373 

## Testing Instructions

* Navigate using tab key to the Request Body tab
* Use the right arrow key tp navigate through tab items 
* Observe correct focus order
